### ISSUE TITLE
Multiple spaces are merged into one

### ISSUE THREAD

**Issue by jhasse** [drone/drone-ui#253]
I've noticed that in the log output multiple spaces are shown as one. Since many terminal programs use spaces for formatting, this messes up their output.

**Comment by bradrydzewski** [comment]
I have not observed this behavior. Can you please provide a simple yaml file that can be used to reproduce the issue?

**Comment by jhasse** [comment]
Sorry I didn't provide one right-away. It's happening for me all the time with everything, so any YAML that prints more than one space works to reproduce. E.g.

```yaml
kind: pipeline
name: default

steps:
- name: build
  image: fedora:29
  commands:
  - echo "   .   .   "
  - echo "---.---.---"
```

Output:

![spaces](https://user-images.githubusercontent.com/80071/51970247-b3099000-2476-11e9-949e-f4854ece383a.png)

**Comment by zwpaper** [comment]
is there any plan for this?

**Comment by tboerger** [comment]
Should be fixed by https://github.com/drone/drone-ui/pull/294

**Comment by bradrydzewski** [comment]
merged #294
